java/Graql.java

```diff
@@ -81,6 +83,17 @@ public static BoundVariable parseVariable(String variable) {
         return parser.parseVariableEOF(variable);
     }
 
+    public static String parseLabel(String label) {
+        String parsedLabel;
+        try {
+            parsedLabel = parser.parseLabelEOF(label);
+        } catch (GraqlException e) {
+            throw GraqlException.of(ILLEGAL_CHAR_IN_LABEL.message(label));
+        }
+        if (!parsedLabel.equals(label)) throw GraqlException.of(ILLEGAL_CHAR_IN_LABEL.message(label)); // e.g: 'abc#123'
```

Comment:
Resulting error message:
`[GQL39] Graql Error: 'person#123' is not a valid Type label. Type labels must start with a letter, and may contain only letters, numbers, '-' and '_'.`